VMT/VMT.md

```diff
@@ -0,0 +1,225 @@
+# Vulnerability Management Process
+
+The Kata Containers Vulnerability Management Team (VMT) is responsible for coordinating the progressive disclosure of a vulnerability.
+
+Members of the team are independent and security-minded folks who ensure that vulnerabilities are dealt with in a timely manner and that downstream stakeholders are notified in a coordinated and fair manner.
+Where a member of the team is employed by a downstream stakeholder, the member does not give their employer prior notice of any vulnerabilities.
+In order to reduce the risk of accidental disclosure of vulnerability in the early stages, membership of this team is intentionally limited to a small number of people.
+
+# Acronyms
+
+To aid reading this document it helps to understand some key acronyms:
+
+| Acronym  | Description                                                     |
+| -------- | --------------------------------------------------------------- |
+| CNA      | [CVE Numbering Authority](https://cve.mitre.org/cve/cna.html)   | 
+| CVE      | [Common Vulnerabilities and Exposures](https://en.wikipedia.org/wiki/Common_Vulnerabilities_and_Exposures) |
+| KCSA     | Kata Containers Security Announcement                           |
+| KCSN     | Kata Containers Security Note                                   |
+| KCST     | Kata Containers Security Team                                   | 
+| PR       | [A GitHub Pull Request](https://help.github.com/articles/about-pull-requests/) |
+| VMT      | The Kata Containers Vulnerability Management Team               |
+
+# Supported versions
+
+> **FIXME** Before we can make statements about which stable versions we fix, we need to define the Kata Containers release and maintenance process.
+
+The VMT coordinates patches fixing vulnerabilities in supported stable branches of Kata Containers, in addition to the master branch (next version under development).
+
+# Process
+
+Each security bug is assigned a VMT *coordinator* (a member from the VMT) that will drive the fixing and disclosure process.
+Here are the steps we follow.
+
+![VMT process workflow](VMT-process.png)
+
+## Reception
+
+A report is received via an email to the [VMT mailing list](mailto:kata-vmt@lists.katacontainers.io).
```

Comment:
A Launchpad was set up so we have the ability to ingest private reports that only go to members of the Architecture Committee. Let's point people here and instruct them to make sure to file for "Private Security" development (aka hitting the private button) https://launchpad.net/katacontainers.io